.github/workflows/pull_request.yaml

```diff
@@ -7,9 +7,28 @@ on:
       - master
 
 jobs:
+  lint:
+    name: cpplint
+    if: contains(join(toJson(github.event.pull_request.labels.*.name)), 'ready-for-testing')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Cpplint
+        run: |
+          set +e -x
+          src_files=$(git --no-pager diff --name-only $(git log --oneline -n 1 | cut -d ' ' -f 1,5) | egrep '.*\.cpp$|.*\.h$|.*\.inl$' | grep -v 'com_vesoft_client_NativeClient.h' | grep -v 'com_vesoft_nebula_NebulaCodec.h')
```

Comment:
why not directly use the pre-commit.sh?

Reply:
I want to do so, but the method to get changed files by git doesn't adapt to github actions since the files has been committed.

Reply:
Fixed it. Thanks.